src/main/java/christmas/constants/EventType.java

```diff
@@ -0,0 +1,29 @@
+package christmas.constants;
+
+public enum EventType {
+    STAR("별", 5000),
+    TREE("트리", 10000),
+    SANTA("산타", 20000);
+
+    private final String name;
+    private final int nedeed;
+
+    EventType(final String name, final int nedeed) {
+        this.name = name;
+        this.nedeed = nedeed;
+    }
+
+    public static String findEventByMoney(final int discountAmonut) {
```

Comment:
```suggestion
    public static String findEventByMoney(final int discountAmount) {
```
금액에 따라 event를 다르게 주는건 충분히 비즈니스 로직으로 보이는데 constance에 넣으신 이유가 궁금해요!